### DISCUSSION TITLE
Do we have any whitepapers?

### DISCUSSION THREAD

**Discussion by yahu1031** [serverpod/serverpod#457] [Q&A] [answered]
I was looking around for the architecture and things. But I can't find any. Is there any whitepapers idea for serverpod? 

@vlidholt @andryHTC

**Comment by AndryHTC** [comment]
No AFAIK, but @vlidholt is the man.

It would be great for pushing performance to the limit

**Comment by vlidholt** [comment, marked as answer]
Currently no whitepapers. May add better documentation for the architecture in the future.
